app/controllers/pages_controller.rb

```diff
@@ -9,6 +9,37 @@ def contact
   def presse
   end
 
+  def carte
+    @vaccination_centers = VaccinationCenter.all.filter { |v| v.confirmed? }.map do |v|
```

Comment:
Avoid Ruby treatment when the filtering can be done by the database.

```ruby
VaccinationCenter.confirmed.map { ... }
```

Reply:
As this may not need real-time updates having a cache around the query can avoid non-necessary load on the database.